apps/src/code-studio/components/progress/UnitOverviewTopRow.jsx

```diff
@@ -124,6 +127,8 @@ class UnitOverviewTopRow extends React.Component {
       ? styles.buttonMarginRTL
       : styles.buttonMarginLTR;
 
+    const showPDFButton = publishedState !== PublishedState.pilot;
```

Comment:
Do we want to include `in_development` here too? PDFs also don't work for in development courses but they're much less visible

Reply:
Thats probably a good idea. I will add